hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->And(vixl::wzr, W(i.s1), MSKTOP(0xff), SetFlags);
```

Comment:
I think `testbi` is supposed to test an immediate `s0` against a register `s1`. `MSKTOP` only masks off the top 32 bits. I think you'll need to mask off the the upper 48 bits. Also for the sake of consistency maybe use `Tst` instead of `And` like `testli` below.